resources/spec.yaml

```diff
@@ -9587,6 +9742,111 @@ definitions:
     type: object
     x-okta-tags:
       - Policy
+  NetworkZone:
+    properties:
+      type:
+        type: string
```

Comment:
Looks like `type` should be an enum with `IP`, `DYNAMIC` as the options.
https://developer.okta.com/docs/reference/api/zones/#zone-object